playwright/integrations/authenticated-chat-with-typing.spec.ts

```diff
@@ -0,0 +1,109 @@
+import fetchOmnichannelConfig from '../utils/fetchOmnichannelConfig';
+import fetchTestPageUrl from '../utils/fetchTestPageUrl';
+import fetchAuthUrl from '../utils/fetchAuthUrl';
+import { test, expect } from '@playwright/test';
+import OmnichannelEndpoints from '../utils/OmnichannelEndpoints';
+
+const testPage = fetchTestPageUrl();
+const omnichannelConfig = fetchOmnichannelConfig('AuthenticatedChatWithTyping');
+const authUrl = fetchAuthUrl('AuthenticatedChatWithTyping');
+
+test.describe('AuthenticatedChat @AuthenticatedChatWithTyping', () => {
+    test('ChatSDK.startChat() should fetch the chat token & perform session init', async ({ page }) => {
```

Comment:
Please update the title of the test case to `ChatSDK.sendTyping()` should not fail`